src/FakeItEasy/Generated/CodeGen/CodeGen.CallbackConfigurationExtensionsGenerator/CallbackConfigurationExtensions.StronglyTyped.cs

```diff
@@ -32,7 +31,7 @@ public static partial class CallbackConfigurationExtensions
                 {
                     ValueProducerSignatureHelper.AssertThatValueProducerSignatureSatisfiesCallSignature(call.Method, actionToInvoke.GetMethodInfo(), NameOfInvokesFeature);
 
-                    actionToInvoke(call.GetArgument<T1>(0));
+                    actionToInvoke(call.GetArgument<T1>(0)!);

```

Comment:
For some reason the generated source triggered a warning about the nullability. I think it's a good change regardless.